thefuck/rules/dnf_install.py

```diff
@@ -0,0 +1,22 @@
+
+
+def get_packages(command):
+    splitted_command = command.script.split()
+    if splitted_command[0] == 'sudo':
+        return splitted_command[3:]
+    return splitted_command[2:]
+
+
+def match(command):
+    typos = ('dfn install', 'dnf istall', 'dfn istall')
```

Comment:
* When a type ```dfn install``` fuck only gives me ```idn_install``` and ```df install```.
* Sorry, i didn't notice ```utils.get_closest```. I will fix it. And yes. There is much more for dnf command, but i wanted to start with a short commit, since this is my first one for this project.